projects/fsastorefrontlib/src/core/my-account/facade/quote.service.ts

```diff
@@ -29,6 +30,8 @@ export class QuoteService {
     protected routingService: RoutingService
   ) {}
 
+  quoteForCompareSource = new BehaviorSubject<InsuranceQuote>(null);
+  quoteForCompare$ = this.quoteForCompareSource.asObservable();
```

Comment:
Add `BehaviorSubject ` to track quote from quote details page when user click on the compare button.